include/realizations/catchment/Simple_Lumped_Model_Realization.hpp

```diff
@@ -4,6 +4,7 @@
 #include <HY_CatchmentArea.hpp>
 #include "LinearReservoir.hpp"
 #include "hymod/include/Hymod.h"
+#include "tshirt/include/Tshirt.h"
```

Comment:
You should not include tshirt code in this file. Make a class like this don't use this file.

Reply:
Thanks, @donaldwj.  Seems I did that several weeks ago, and I'm not exactly sure why.